### ISSUE TITLE
Adding mean to plot about mean_uncertainty

### ISSUE THREAD

**Issue by ju-arroyom** [erikbern/uncertainty#1]
Hi Erik,

I really enjoyed your blog post about uncertainty estimates, and I would like to make a small suggestion that I think would help clarify a point you make in the blog.

In your first plot for the confidence intervals of the time series, you emphasize that the plot shows uncertainty of the _mean_ and that this should not be confused with the _distribution of the data_ itself.

I think it would be helpful to add the line with the value of the mean, so that you can see how the confidence intervals are always around the mean.

![screen shot 2018-10-10 at 1 50 59 pm](https://user-images.githubusercontent.com/25286084/46758906-a973d680-cc93-11e8-8b9a-3580c5234645.png)

**Comment by erikbern** [comment]
yeah i agree that it would make sense to do that. i might make that update!

**Comment by ju-arroyom** [comment]
Would you like me to submit a PR?

**Comment by erikbern** [comment]
no it's ok. i can do it at some point.
